docs/user-guides/workspace-access/jetbrains.md

```diff
@@ -17,7 +17,7 @@ IDEs are supported for remote development:
 
 JetBrains Gateway is a compact desktop app that allows you to work remotely with
 a JetBrains IDE without even downloading one.
-[See JetBrains' website to learn about and Gateway.](https://www.jetbrains.com/remote-development/gateway/)
+[See JetBrains's website to learn about and Gateway.](https://www.jetbrains.com/remote-development/gateway/)
```

Comment:
As the last letter is an `s` we don't need an extra apostrophe s

Reply:
[Google Developer Documentation Style Guide](https://developers.google.com/style/possessives) suggests avoiding it altogether, so I adjusted this line